CMakeLists.txt

```diff
@@ -85,6 +85,15 @@ include_directories(
   ${CMAKE_CURRENT_SOURCE_DIR}/include
   ${CMAKE_CURRENT_BINARY_DIR}/include)
 
+############## Code Coverage #########################
+option(ENABLE_COVERAGE "Enable coverage reporting." OFF)
+
+if (ENABLE_COVERAGE)
+  include(CodeCoverage)
+  APPEND_COVERAGE_COMPILER_FLAGS()
```

Comment:
```suggestion
  append_coverage_compiler_flags()
```

My PR upstream will lowercase these in the module file, as well — current CMake norms are that `ALLCAPS_IS_FOR_VARIABLES` while `functions_use_lowercase()`.